packages/icon-explorer/src/App.tsx

```diff
@@ -35,52 +36,31 @@ const App = () => {
     setState({ view: 'IconSet', iconName, iconStyle });
   };
 
```

Comment:
not sure what exactly is going on, but android back button minimizes the icon explorer app for me, instead of going from the detail view (individual icon family) to the list of icon families.

Reply:
> not sure what exactly is going on, but android back button minimizes the icon explorer app

Ahh I accidentally removed the hardware back button support when tidying up something else